api/assessment/assessment.go

```diff
@@ -40,6 +40,12 @@ var (
 	ErrMetricConfigurationTargetValueMissing = errors.New("target value in metric data is missing")
 )
 
+const (
+	DefaultNonCompliantMessage = "The metric has assessed that the resource contains properties that are not compliant to the target value."
+	DefaultCompliantMessage    = "The result of the metric shows that the evidence is compliant to the target value."
+	AdditionalDetailsMessage   = "Additional details can be found in the comparison below."
```

Comment:
Sure, that the details are always below? Or are they in the field ComparisonResult?

Reply:
Well they are below in the UI ;)

Reply:
I find that difficult. If someone changes the UI view and doesn't update it accordingly, no one will know where to find anything. Additionally, if it's retrieved using curl, it probably isn't below. 

I think something like "Additional details can be found in the comparison" or "comparison view/part/field"  better.